### PR TITLE
update schedule samples for new scheduler

### PR DESCRIPTION
**Pull request by ig-ra**

### CODE REVIEW

File: scheduler/program.star [hunk 1]
```diff
@@ -10,32 +10,38 @@ Starlark is a dialect of Python (see https://bazel.build/rules/language).
...
-
-    my_slack.chat_post_message(SLACK_CHANNEL, msg)
+load("@github", "my_github")
+
+# Set in ""autokitteh.yaml"
+load("env", "SLACK_CHANNEL", "GITHUB_OWNER", "GITHUB_REPO")
+
+def on_cron_trigger():
+    """An autokitteh cron schedule was triggered."""
+    #my_slack.chat_post_message(SLACK_CHANNEL, "daily reminder about open PRs")
```
Comment (daabr) [inline scheduler/program.star:20]:
```suggestion
    # my_slack.chat_post_message(SLACK_CHANNEL, "Daily reminder about open PRs")
```

File: scheduler/program.star [hunk 2]
```diff
@@ -10,32 +10,38 @@ Starlark is a dialect of Python (see https://bazel.build/rules/language).
...
+
+    # fetch PRs
```
Comment (daabr) [inline scheduler/program.star:22]:
```suggestion
    # Fetch all PRs.
```

File: scheduler/program.star [hunk 3]
```diff
@@ -10,32 +10,38 @@ Starlark is a dialect of Python (see https://bazel.build/rules/language).
...
+    prs = my_github.list_pull_requests(owner=GITHUB_OWNER, repo=GITHUB_REPO, state="open")
+
+    # filter, skip DRAFT or WIP PRs
```
Comment (daabr) [inline scheduler/program.star:25]:
```suggestion
    # Filter: skip DRAFT and WIP PRs.
```

File: scheduler/program.star [hunk 4]
```diff
@@ -10,32 +10,38 @@ Starlark is a dialect of Python (see https://bazel.build/rules/language).
...
+            continue
+        active_prs.append(pr)
+
+    now = time.now()
+    good_updated_at = time.from_timestamp(now.unix - 1 * 60 * 60 * 24) # a day
+    good_opened_at = time.from_timestamp(now.unix - 1 * 60 * 60 * 24 * 4) # 4 days
+
+    for pr in active_prs:
+        s = ""
+        # check whether this PR is stalled - either opened or updated a long ago
```
Comment (daabr) [inline scheduler/program.star:38]:
```suggestion
        # Check whether this PR is stalled - either opened or updated a long ago.
```

**Review by daabr — COMMENTED** [review]
Various nits, but overall this is good.

File: scheduler/README.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
 # Scheduler (Cron) Sample Project
 
-This sample project demonstrates AutoKitteh's integration with a cron-like
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/README.md:3]: I think the decision was to use `autokitteh` and not `AutoKitteh`

File: scheduler/README.md [hunk 6]
```diff
@@ -17,21 +17,18 @@ environment in the [`autokitteh.yaml`](./autokitteh.yaml) manifest file.
 2. Create AutoKitteh connection tokens
 
    1. Open a browser, and go to the AutoKitteh server's URL
-   2. Create Scheduler (Cron) and Slack connections, and copy the resulting
-      tokens
+   2. Create Slack connections, and copy the resulting tokens
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/README.md:20]: resolving then

File: scheduler/autokitteh.yaml [hunk 7]
```diff
@@ -1,28 +1,38 @@
...
+
 # Before applying this file:
-# - Set the "SLACK_CHANNEL" value in the "vars" section
-# - Create AutoKitteh connection tokens for the Scheduler (Cron) and Slack,
-#   and replace the "TODO" lines below
+# - Set variables SLACK_CHANNEL, GITHUB_OWNER and GITHUB_REPO in the
+#   manifest "vars" section
+# - Create and initialize autokitteh connection for Slack and Github
+
+# Schedule specification format is one from the follows:
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/autokitteh.yaml:13]: done

File: scheduler/program.star [hunk 8]
```diff
@@ -1,7 +1,7 @@
-"""This program demonstrates AutoKitteh's scheduler (cron) integration.
+"""This program demonstrates autokitteh's scheduler abilities.
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/program.star:1]: done

File: scheduler/autokitteh.yaml [hunk 9]
```diff
@@ -1,28 +1,38 @@
 # This YAML file is a declarative manifest that describes the setup
 # of an AutoKitteh sample project that demonstrates integration with
 # a cron-like scheduler.
-#
+
+# The program will scan daily specific github repo for stalled PRs
+# and, if found, will send a message specific Slack channel
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/autokitteh.yaml:6]: done

File: scheduler/program.star (hunk 2 above)
Comment (ig-ra) [reply scheduler/program.star:22]: done, except the case :) partial sentences shouldn't be capitalized

Comment (ig-ra) [reply scheduler/program.star:22]: done, except the case :) partial sentences shouldn't be capitalized

File: scheduler/program.star (hunk 4 above)
Comment (ig-ra) [reply scheduler/program.star:38]: hmm.. will skip it. I still believe that single line comment is usually a partial sentence and doesn't require capitalization

File: scheduler/program.star (hunk 3 above)
Comment (ig-ra) [reply scheduler/program.star:25]: done (except caps)

File: scheduler/program.star (hunk 1 above)
Comment (ig-ra) [reply scheduler/program.star:20]: done

File: scheduler/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/README.md:3]: fixed

File: scheduler/README.md [hunk 10]
```diff
@@ -17,21 +17,18 @@ environment in the [`autokitteh.yaml`](./autokitteh.yaml) manifest file.
...
+3. Via the `ak` CLI tool, or the autokitteh VS Code extension, deploy the
    `autokitteh.yaml` manifest file
 
 ## Connection Notes
 
 Information about cron schedules:
 
-- [Cron expression format ("\* \* \* \* \*")](https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format)
+- [Cron expression format ("* * * * ")](https://pkg.go.dev/github.com/robfig/cron#hdr-CRONExpression_Format)
+- [Cron extended expression format ("* * * * * * ")](https://pkg.go.dev/github.com/robfig/cron/v3#hdr-AlternativeFormats)
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/README.md]: done

File: scheduler/autokitteh.yaml [hunk 11]
```diff
@@ -1,28 +1,38 @@
...
-# - Set the "SLACK_CHANNEL" value in the "vars" section
-# - Create AutoKitteh connection tokens for the Scheduler (Cron) and Slack,
-#   and replace the "TODO" lines below
+# - Set variables SLACK_CHANNEL, GITHUB_OWNER and GITHUB_REPO in the
+#   manifest "vars" section
+# - Create and initialize autokitteh connection for Slack and Github
+
+# Schedule specification format is one from the follows:
+# - standard 5 star cron spec
+# - extended 7 star cron spec (seconds, minutes, etc..)
```
[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/autokitteh.yaml:15]: up to 7

[the remark this answers is not in the document]
Comment (ig-ra) [reply scheduler/autokitteh.yaml:15]: seconds and year, I think

**Review by daabr — APPROVED** [review]

**Pull request merged**